package.json

```diff
@@ -29,8 +29,8 @@
   ],
   "scripts": {
     "test": "npm run lint && metatests test/",
-    "lint": "eslint --ignore-path .gitignore . && prettier -c \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
-    "fmt": "prettier --write \"**/*.js\" \"**/*.json\" \"**/*.md\" \".*rc\" \"**/*.yml\"",
+    "lint": "eslint \"**/*\" && prettier -c \"*\"",
+    "fmt": "prettier --write \"*\" && eslint --fix \"**/*\"",
```

Comment:
While we are at it, should we perhaps reorder them according to `lint` command (eslint first, then prettier)?